app/src/app.tsx

```diff
@@ -10,14 +10,15 @@ import SubmissionTextBox from "./components/submissionPage/SubmissionTextBox";
 import {
     getCurrentUserName,
     onAuthStateChanged,
-    ProfileData,
-} from "./firebase/AuthService";
+} from "./firebase/users/UserService";
+import { UserProfile } from "./firebase/users/UserProfile";
+import YourSnaps from "./components/submissionPage/YourSnaps";
```

Comment:
It's a bit odd to import from within a page folder straight into the app. Is there a plan to refactor this?

Reply:
Yes I think refactoring is necessary but perhaps at a subsequent stage?